com.github.alexkdeveloper.desktop-files-creator.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id" : "com.github.alexkdeveloper.desktop-files-creator",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "desktop-files-creator",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=home:ro",
+        "--filesystem=xdg-data/applications:create"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "Desktop Files Creator",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/alexkdeveloper/desktop-files-creator.git",
```

Comment:
You should consider configuring https://github.com/flathub/flatpak-external-data-checker/ to automate updates but it's not a blocker.